odoo/addons/test_access_rights/tests/test_res_config.py

```diff
@@ -0,0 +1,30 @@
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestResConfig(TransactionCase):
+    def test_apply_groups(self):
```

Comment:
I think the use of "standard" groups, and the *not* use of the `_apply_group` method directly makes this test a lot more confusing than it needs to be.

Using `group_no_one` is especially problematic due to the magic surrounding it in multiple places (e.g. it's special cased in `user_has_groups`).